### PR TITLE
dual-stack blog for Chinese

### PR DESCRIPTION
**Pull request by zhlsunshine**
Please provide a description for what this PR is for.

This PR is the translation of [PR#12253](https://github.com/istio/istio.io/pull/12253)

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by windsonsea** [comment]
some errors are included, which you can check the [error details](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/12449/lint_istio.io/1613420132950872064).

```
>> 14 spelling errors found in 757 files
To learn how to address spelling errors, please see https://istio.io/about/contribute/build/#test-your-changes
./zh/blog/2023/dual-stack/index.md:11: MD002 First header should be a top level header
./zh/blog/2023/dual-stack/index.md:[48](https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio.io/12449/lint_istio.io/1613420132950872064#1:build-log.txt%3A48): MD009 Trailing spaces
./zh/blog/2023/dual-stack/index.md:48: MD034 Bare URL used
```

File: content/zh/blog/2023/dual-stack/index.md
Comment (rootsongjc) [inline content/zh/blog/2023/dual-stack/index.md:3]:
```suggestion
description: "双栈 Kubernetes 集群的实验版本和主干分支 Alpha 版本支持。"

```

File: content/zh/blog/2023/dual-stack/index.md
Comment (rootsongjc) [inline content/zh/blog/2023/dual-stack/index.md:13]:
```suggestion
对于 Isito 双栈支持的工作花费了比预期更长的时间，而我们也还有很多关于双栈的工作需要继续。最初这项工作基于 F5 的设计实现展开，由此我们创建了 [RFC](https://docs.google.com/document/d/1oT6pmRhOw7AtsldU0-HbfA0zA26j9LYiBD_eepeErsQ/edit?usp=sharing) ，社区根据该设计文档展开了广泛的讨论。值得注意的是，社区对此方案存在对内存和性能方面的顾虑，并且希望这些问题能够在实现之前被解决，这也引起了我们对最初设计方案的反思。

```

File: content/zh/blog/2023/dual-stack/index.md
Comment (zhlsunshine) [reply content/zh/blog/2023/dual-stack/index.md:3]: okay, done!

File: content/zh/blog/2023/dual-stack/index.md
Comment (zhlsunshine) [reply content/zh/blog/2023/dual-stack/index.md]: Hi @zirain, it seems that your CLA caused the job failed, could you please help to fix it? Thanks!

**Comment by windsonsea** [comment]
The lint error may inherit from the en #12253. This PR may need to wait for en merged, right?

**Comment by zhlsunshine** [comment]
> The lint error may inherit from the en #12253. This PR may need to wait for en merged, right?

Hi @windsonsea, yeah, you are right, I need to wait for the en PR#12253 get merged. Thanks!

File: content/zh/blog/2023/dual-stack/index.md
Comment (zirain) [inline content/zh/blog/2023/dual-stack/index.md:16]:
```suggestion
随着持续深入的探索，在重新评估技术方案的同时，我们创建了一个新分支 [experimental-dual-stack]({{< github_raw >}}/tree/experimental-dual-stack)来参考最初的设计来实现和验证 Istio 双栈的实现方案。

```

Comment (zirain) [reply content/zh/blog/2023/dual-stack/index.md:16]: this will fix lint

File: content/zh/blog/2023/dual-stack/index.md
Comment (zhlsunshine) [reply content/zh/blog/2023/dual-stack/index.md:4]: okay, got it

**Review by zirain — APPROVED** [review]
LGTM, add `DNM` until https://github.com/istio/istio.io/pull/12253 merged

**Comment by zirain** [comment]
@zhlsunshine can you sync this to en blog?

**Comment by zhlsunshine** [comment]
> @zhlsunshine can you sync this to en blog?

@zirain Yes, thank you for the remind,  re-submitted.

File: .spelling
Comment (zirain) [inline .spelling:403]: still need this?

**Comment by zirain** [comment]
@rootsongjc would you like to take another look?

File: content/zh/blog/2023/experimental-dual-stack/index.md
Comment (windsonsea) [inline content/zh/blog/2023/experimental-dual-stack/index.md:98]: The indentations on line 36-98 are not correct now. Ident 3 spaces please.

Comment (zhlsunshine) [reply content/zh/blog/2023/experimental-dual-stack/index.md]: Hi @windsonsea  done

Comment (zhlsunshine) [reply content/zh/blog/2023/experimental-dual-stack/index.md]: but I encountered below lint error when I Ident 3 spaces between line# 36-98

File: .spelling
Comment (zhlsunshine) [reply .spelling:403]: removed

File: content/zh/blog/2023/experimental-dual-stack/index.md
Comment (zhlsunshine) [reply content/zh/blog/2023/experimental-dual-stack/index.md:98]: `Text blocks need to not be indented, or indented by a multiple of 4 spaces`
So I think it may be unnecessary to indent here.

**Comment by zirain** [comment]
@windsonsea feel free to remove `DNM` if you're ok.

File: content/zh/blog/2023/experimental-dual-stack/index.md
Comment (windsonsea) [inline content/zh/blog/2023/experimental-dual-stack/index.md:64]:
```suggestion
    {{< text bash >}}
    $ kubectl create namespace dual-stack
    $ kubectl create namespace ipv4
    $ kubectl create namespace ipv6
    {{< /text >}}
```

File: content/zh/blog/2023/experimental-dual-stack/index.md
Comment (windsonsea) [inline content/zh/blog/2023/experimental-dual-stack/index.md:73]:
```suggestion
    {{< text bash >}}
    $ kubectl label --overwrite namespace default istio-injection=enabled
    $ kubectl label --overwrite namespace dual-stack istio-injection=enabled
    $ kubectl label --overwrite namespace ipv4 istio-injection=enabled
    $ kubectl label --overwrite namespace ipv6 istio-injection=enabled
    {{< /text >}}
```

File: content/zh/blog/2023/experimental-dual-stack/index.md
Comment (windsonsea) [inline content/zh/blog/2023/experimental-dual-stack/index.md:81]:
```suggestion
    {{< text bash >}}
    $ kubectl apply --namespace dual-stack -f {{< github_file >}}/samples/tcp-echo/tcp-echo-dual-stack.yaml
    $ kubectl apply --namespace ipv4 -f {{< github_file >}}/samples/tcp-echo/tcp-echo-ipv4.yaml
    $ kubectl apply --namespace ipv6 -f {{< github_file >}}/samples/tcp-echo/tcp-echo-ipv6.yaml
    {{< /text >}}
```

File: content/zh/blog/2023/experimental-dual-stack/index.md
Comment (windsonsea) [inline content/zh/blog/2023/experimental-dual-stack/index.md:87]:
```suggestion
    {{< text bash >}}
    $ kubectl apply -f {{< github_file >}}/master/samples/sleep/sleep.yaml
    {{< /text >}}
```

File: content/zh/blog/2023/experimental-dual-stack/index.md
Comment (windsonsea) [inline content/zh/blog/2023/experimental-dual-stack/index.md:98]:
```suggestion
    {{< text bash >}}
    $ kubectl exec -it "$(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}')" -- sh -c "echo dualstack | nc tcp-echo 9000"
    hello dualstack
    $ kubectl exec -it "$(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}')" -- sh -c "echo ipv4 | nc     tcp-echo.ipv4 9000"
    hello ipv4
    $ kubectl exec -it "$(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}')" -- sh -c "echo ipv6 | nc tcp-echo.ipv6 9000"
    hello ipv6
    {{< /text >}}

```

**Review by windsonsea — COMMENTED** [review]
Yes, we may need to indent 4 spaces by following the message tips. Otherwise, the ordered list will be like [this](https://github.com/istio/istio.io/blob/0c5541213f78f7d08bf388cb8e17ec3eab7236e2/content/zh/blog/2023/experimental-dual-stack/index.md#%E4%BD%BF%E7%94%A8%E5%8F%8C%E6%A0%88%E7%9A%84%E5%BF%AB%E9%80%9F%E5%AE%9E%E9%AA%8C).

**Comment by zhlsunshine** [comment]
Hi @windsonsea,  there are the same lint errors both I changed the indent and committed your suggest comments.

File: content/zh/blog/2023/experimental-dual-stack/index.md
Comment (windsonsea) [inline content/zh/blog/2023/experimental-dual-stack/index.md:59]:
```suggestion
    * `dual-stack`: `tcp-echo` will listen on both an IPv4 and IPv6 address.
    * `ipv4`: `tcp-echo` will listen on only an IPv4 address.
    * `ipv6`: `tcp-echo` will listen on only an IPv6 address.

```
These need localization too.

**Review by windsonsea — COMMENTED** [review]
Normally, we only need to follow the en style, including those indentations.

If still has problem, you can restore it to be free of lint-error, I can handle more revisions after your merging :)

**Comment by zhlsunshine** [comment]
> Normally, we only need to follow the en style (copy-paste those code), including those indentations.
> 
> If still has problem, you can restore it to be free of lint-error, I can handle more revisions after your merging :)

Hi @windsonsea, okay, please let me know how to co-operate with you if possible.

**Comment by windsonsea** [comment]
I can't edit your file, don't know why.

All right, you can try to enable the CI success, merge it, and I will handle it soon.

image

**Comment by zhlsunshine** [comment]
Hi @windsonsea 

> All right, you can try to enable the

I do not know why,  I'm trying to using the English version to keep the original format. Let me submit it again. Thanks!

**Comment by zhlsunshine** [comment]
Hi @windsonsea, it's okay if I use the original format as you mentioned, thanks!

**Comment by zhlsunshine** [comment]
> In response to a cherrypick label: new issue created for failed cherrypick: #12884

Hi @windsonsea I noticed that there still is a cherrypick issue, so shall we do anything for it?

**Pull request merged**
